tests/end_to_end/target_snowflake/__init__.py

```diff
@@ -0,0 +1,54 @@
+import os
+import shutil
+import unittest
+from pathlib import Path
+
+from tests.end_to_end.helpers import assertions, tasks
+from tests.end_to_end.helpers.env import E2EEnv
+
+TEST_PROJECTS_DIR_PATH = "tests/end_to_end/test-project"
+USER_HOME = os.path.expanduser("~")
+CONFIG_DIR = os.path.join(USER_HOME, ".pipelinewise")
+
+
+class TargetSnowflake(unittest.TestCase):
+    def setUp(self) -> None:
+        super().setUp()
```

Comment:
I am a little confused here!
why `super().setUp()` is needed here?

Reply:
It was generated and I didn't remove it. Let me know if you would like it to be removed.

Reply:
I think it would be nicer and more clean if you remove it